fe/src/main/cup/sql_parser.cup

```diff
@@ -1515,6 +1521,15 @@ partition_key_desc ::=
     :}
     ;
 
+fixed_partition_key_desc ::=
+    LBRACKET string_list:keys RPAREN
+    {:
+        if (keys.size() != 2) {
```

Comment:
I think the proper format is:
`[("low1", "low2"), ("up1", "up2"))`

And should also support :
`[("low1", "low2", ..), ("up1"))`, and the default value should be filled with min value

Do we support MAXVALUE?